integration/Makefile

```diff
@@ -14,3 +14,6 @@ auction_test:
 
 echo_test:
 	./echo_test.sh
+
+two_test:
```

Comment:
Don't you want to include that also in the main target so it is called by default?

Reply:
oh I missed that! I also rename `two_test` to `deployment_test`